src/main/webapp/app/exam/manage/exam-management.component.html

```diff
@@ -75,7 +77,7 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.examManagement.title">Exam
                                     <span class="d-none d-md-inline">{{ 'artemisApp.examManagement.studentExams' | artemisTranslate }}</span>
                                 </a>
                                 <a
-                                    *ngIf="(course.isAtLeastTutor && examHasFinished(exam)) || course.isAtLeastInstructor"
+                                    *ngIf="!exam.testExam && ((course.isAtLeastTutor && examHasFinished(exam)) || course.isAtLeastInstructor)"
```

Comment:
Can it be problematic, that `exam.testExam` is undefined?
If `testExam` is undefined, `exam.testExam` evaluates to false and `!exam.testExam` evaluatest to true.

Reply:
exam.testExam is always defined, as it is enforced during creation / update of an exam, and it is always sent to the client when getting an exam from the client.
Thanks for the good catch, we'll consider changing the entity model from a ?: boolean to a : boolean, as we also use a boolean on the server-side (instead of Boolean)